payments-ui-core/src/main/java/com/stripe/android/ui/core/StripeTheme.kt

```diff
@@ -1,83 +1,144 @@
 package com.stripe.android.ui.core
 
+import android.content.Context
+import android.content.res.Configuration.UI_MODE_NIGHT_MASK
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
 import androidx.annotation.RestrictTo
 import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.Colors
 import androidx.compose.material.MaterialTheme
-import androidx.compose.material.darkColors
 import androidx.compose.material.lightColors
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.staticCompositionLocalOf
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.unit.sp
 
-private val Green400 = Color(0xFF3CB043)
-private val Green800 = Color(0xFF234F1E)
-
-private val Yellow400 = Color(0xFFF6E547)
-private val Yellow700 = Color(0xFFF3B711)
-private val Yellow800 = Color(0xFFF29F05)
-
-private val Blue200 = Color(0xFF9DA3FA)
-private val Blue500 = Color(0xFF0540F2)
+internal val LocalFieldTextStyle = TextStyle.Default.copy(
+    fontFamily = FontFamily.SansSerif,
+    fontSize = 14.sp
+)
 
-private val Red300 = Color(0xFFEA6D7E)
-private val Red800 = Color(0xFFD00036)
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class PaymentSheetConfigColors(
```

Comment:
If these are PaymentSheet-specific then they should be in the paymentsheet module. It looks like they're not, except for appBarIcon. So I think they can be renamed to something like PaymentsUI instead of PaymentSheet. Then the PaymentSheet theme can be an extension of this. WDYT?

Reply:
Thinking about other modules reusing this and also setting their desired values, for example Link.

Reply:
I think appBarIcon color is generic enough, good chance we have another app bar type ui in the future that I don't think it warrants moving it out. I did rename these to `Payments***` rather than `PaymentSheet***`. I agree it's clearer. 

I don't want to add extra scope to this PR so it's easy to read. I think we'll probably get the ability for other modules to reuse these values when they become mutable with the appearance API. Unless there's a ton of FUD I'm going to keep the functionality of this PR as it is now.